src/test/resources/jsonld/HT014078228.json

```diff
@@ -179,30 +172,5 @@
     "label" : "Webseite der hbz-Ressource HT014078228"
   },
   "@context" : "http://lobid.org/resources/context.jsonld",
-  "id" : "http://lobid.org/resources/HT014078228#!",
-  "spatial" : [ {
-    "focus" : {
-      "id" : "http://www.wikidata.org/entity/Q1374344",
-      "type" : [ "http://www.wikidata.org/entity/Q153026" ]
-    },
-    "id" : "https://nwbib.de/spatial#Q1374344",
-    "type" : [ "Concept" ],
-    "label" : "Euregio Rhein-Maas-Nord",
-    "source" : {
-      "id" : "https://nwbib.de/spatial",
-      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
-    }
-  }, {
-    "focus" : {
-      "id" : "http://www.wikidata.org/entity/Q1374348",
-      "type" : [ "http://www.wikidata.org/entity/Q153026" ]
-    },
-    "id" : "https://nwbib.de/spatial#Q1374348",
-    "type" : [ "Concept" ],
-    "label" : "Euregio Rhein-Waal",
-    "source" : {
-      "id" : "https://nwbib.de/spatial",
-      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
-    }
-  } ]
```

Comment:
In this one, we seem to be losing the `spatial` field.

Reply:
Also maybe re: new data. Could you have a closer look @TobiasNx ?